kokoro/scripts/utils/louhi.sh

```diff
@@ -28,12 +28,11 @@ function populate_env_vars_from_louhi_tag_if_present() {
   # Example value: louhi/2.46.0/shortref/windows/x86_64/start
   if [[ -n "${_LOUHI_TAG_NAME:-}" ]]; then
     local -a _LOUHI_TAG_COMPONENTS=(${_LOUHI_TAG_NAME//\// })  
-    RELEASE_ID="${_LOUHI_TAG_COMPONENTS[2]}"
```

Comment:
It might be more readable to still have a `RELEASE_ID` variable, but just change the value, i.e.:
```sh
    RELEASE_ID="${_LOUHI_EXECUTION_ID}"
    …
    export REPO_SUFFIX="${RELEASE_ID}"
```

Reply:
This PR stops using `RELEASE_ID` entirely